casbin/model/scope_config.cpp

```diff
@@ -278,6 +278,23 @@ void EvalNoResult(Scope scope, std::string expression){
     duk_eval_string_noresult(scope, expression.c_str());
 }
 
+void DeletePropFromObject(Scope scope, std::string object_name, std::string prop_name) {
+    if (duk_get_global_string(scope, object_name.c_str())) {
+        duk_del_prop_string(scope, -1, prop_name.c_str());
+    }
+}
+
+void CleanScope(Scope scope) {
+    std::vector<std::string> props = {"obj", "sub", "act"};
```

Comment:
The props depend on config. It should be dynamic.

Reply:
Thanks for your review. I move `clean_scope` as a private member function in `enforecer`, because it based on model's config. And I exposed `DeletePropFromObject` for `Enforcer` to edit object in `duktape`. In this way, the clean step will be dynamic.